src/situations/commun/infra/synchronisateur.js

```diff
@@ -0,0 +1,32 @@
+export default class Synchronisateur {
+  constructor (registreUtilisateur) {
+    this.registreUtilisateur = registreUtilisateur;
+    this.enCoursDeSynchronisation = false;
+  }
+
+  recupereReseau (event) {
+    if (this.enCoursDeSynchronisation === true) { return; }
```

Comment:
C'est juste une question de style mais vraiment je préfèrerait qu'on écrive `if (this.enCoursDeSynchronisation) { return; }`